src/Symfony/Component/ClassLoader/ClassCollectionLoader.php

```diff
@@ -43,12 +43,12 @@ public static function load($classes, $cacheDir, $name, $autoReload, $adaptive =
 
         self::$loaded[$name] = true;
 
-        $declared = array_merge(get_declared_classes(), get_declared_interfaces());
```

Comment:
when $adaptive is false, which is the case in all usages in Symfony,
and when the cache is warm
this $declared variable is currently computed but never used.